### ISSUE TITLE
Inconsistency in space and return key for button selection

### ISSUE THREAD

**Issue by Changsuwan** [SAP/spartacus#11557]
For "Add to cart", "Select", "Add" button, keys for selection should be consistent. 

For example:
In the detail page "configure" button => only the return key works
"Add to cart button" in the configurator => only the return key works
select, add, and configure button => space and return key work
Also consider group navigation

**Comment by ChristophHi** [comment]
Assigning this to @Changsuwan as changes were requested...Please, if you have covered all concerns, assign it to Larisa for another review
